user_guide/data_management_workflows.rst

```diff
@@ -0,0 +1,105 @@
+.. _user-guide-workflow-management-basics:
+
+Following a Workflow
+====================
+
+In the process of the work with the customers information saved in the CRM system can be changed: carts turn 
+into orders, what seemed to be a potential opportunity appears not to be worth effort or tuns into a successful deal, 
+customer support requests are processed and closed, and so on.
+These and other processes are reflected in OroCRM with creation of new records, change of record properties and 
+creation of new relations. (For example - a new lead appeared, and while processed, it was related to a customer and a 
+contact, and turned into a new opportunity. 
+
+In many cases success of the business depends a lot on the unity of the procedures followed all through the company.
+How many times can you call a customer? Can you close an opportunity as lost without an explanation? What should you do 
+if a customer has added some goods to the cart but hasn't ordered them?
+
+In OroCRM there can be defined workflows, that system users should follow in order to process entities. For example, a 
+workflow for processing an abandoned cart (when a customer has added at least one item but has not purchased 
+anything) will allow a manager to convert the cart into an order at any moment, but will not let set a cart to the 
+“abandoned” status until the customer has been contacted successfully.
+
+In a technical language, a a workflow is a sequence of industrial, administrative or other processes applied to a 
+piece of work from the initiation to completion that can be defined for records of specific type.
+
+This article is aimed to provide the understanding of workflows and ways to follow them. The way to create workflows is 
+described in the part of the guide sharpened for the system administrators.
+
+Workflow Components 
+-------------------
+
+Steps and Transitions
+^^^^^^^^^^^^^^^^^^^^^
+Each of the processes or actions applied to a record is a *workflow transition* and the state of this record before or
+after a transition is a *workflow step*.
```

Comment:
Edit: Each of the processes or actions applied to a record is called a *workflow transition*, and the state of this record before or after a transition is called a *workflow step*.